garage/envs/mujoco/gather/gather_env.py

```diff
@@ -438,14 +430,12 @@ def get_ori(self):
         successful, falls back to the default based on the ORI_IND specified in
         Maze (not accurate for quaternions)
         """
-        obj = self.wrapped_env
-        while not hasattr(obj, 'get_ori') and hasattr(obj, 'wrapped_env'):
```

Comment:
uhh does this still implement the behavior in the comment? i know it's a gross implementation, but we shouldn't break functionality to aesthetics.

Reply:
I'm not sure. I can put it back in but since we're inheriting from gym.Wrapper I'm not sure if `hasattr(obj, 'env')` will still work.

Reply:
i mean, if it doesn't, then we need to figure out how to implement the same logic with the new wrapper. i don't think the original author put this here for just for fun.